### PR TITLE
gh-96165: Clarify omitting the FROM clause in SQLite queries.

### PR DESCRIPTION
**Pull request by felixxm**
* Issue: gh-96165

----
:books: Documentation preview :books:: https://cpython-previews--106513.org.readthedocs.build/

### CODE REVIEW

**Comment by erlend-aasland** [comment]
Thanks, Mariusz! I wonder if we should put these hints in their own how-to.

@CAM-Gerlach and @ezio-melotti, what do you think?

File: Doc/library/sqlite3.rst [hunk 1]
```diff
@@ -2519,6 +2519,13 @@ Queries now return :class:`!Row` objects:
    >>> row["RADIUS"]  # Column names are case-insensitive.
    6378
 
+.. note::
+
+    The ``FROM`` clause can be ommitted in the ``SELECT`` statement, as in the
```
Comment (ezio-melotti) [inline Doc/library/sqlite3.rst]:
```suggestion
    The ``FROM`` clause can be omitted in the ``SELECT`` statement, as in the
```

**Comment by ezio-melotti** [comment]
I remember noticing the "unusual" query with no `FROM` clause, and I figured out how it worked from the output.  Having a more explicit explanation just after the example makes things clearer.  If the note is moved elsewhere it might be harder to find unless you add a link to it.  I'm also not entirely sure about what else you were thinking to add in the how-to.

Regarding the note itself, I wonder if the wording could be simplified a bit, since in just two sentences it talks about clauses, statements, row, columns, expressions, literals, and aliases.  Even though it's correct and understandable, if the reader is not familiar with all the terms, it might take a couple of re-reads to understand it.

**Comment by felixxm** [comment]
> I wonder if we should put these hints in their own how-to.

I'm not sure if it's worth. For example, in Django we try to avoid duplicating of 3rd-party documentation.

File: Doc/library/sqlite3.rst (hunk 1 above)
Comment (felixxm) [reply Doc/library/sqlite3.rst]: Corrected, thanks :+1:

**Comment by felixxm** [comment]
> Regarding the note itself, I wonder if the wording could be simplified a bit, since in just two sentences it talks about clauses, statements, row, columns, expressions, literals, and aliases. Even though it's correct and understandable, if the reader is not familiar with all the terms, it might take a couple of re-reads to understand it.

We could use `constant values` instead of `literals`, and `names` instead of `aliases` to reduce the number of terms, e.g.
```diff
diff --git a/Doc/library/sqlite3.rst b/Doc/library/sqlite3.rst
index f6a8714519..6d9b1c22ce 100644
--- a/Doc/library/sqlite3.rst
+++ b/Doc/library/sqlite3.rst
@@ -2523,7 +2523,7 @@ Queries now return :class:`!Row` objects:
 
     The ``FROM`` clause can be omitted in the ``SELECT`` statement, as in the
     above example. In such cases, SQLite returns a single row with columns
-    defined by expressions, e.g. literals, with the given aliases
+    defined by expressions, e.g. constant values, with the given names
     ``expr AS alias``.
 
 You can create a custom :attr:`~Cursor.row_factory`
```

On the other hand, the term `literal` is already used multiple times in this page :shrug:

**Comment by erlend-aasland** [comment]
I think 'literal' is fine in programming language docs ;)

**Review by erlend-aasland — APPROVED** [review]

**Comment by felixxm** [comment]
@erlend-aasland Thanks :+1:

**Pull request merged**